protocol/consistencychecks.go

```diff
@@ -100,6 +100,50 @@ func (cc *ConsistencyChecks) HandleResponse(requestType int, msg *Response,
 	return CheckPassed
 }
 
+// handleDirectorySTRs is supposed to be used by CONIKS clients to
+// handle auditor responses, and by CONIKS auditors to handle directory responses.
```

Comment:
I've started this in https://github.com/coniks-sys/coniks-go/tree/generic-auditor What do you think?